src/app/Fake.Runtime/CompileRunner.fs

```diff
@@ -104,12 +106,16 @@ let compile (context:FakeContext) outDll =
     if (destinationFile.Exists) then destinationFile.Delete()
 
     let co = context.Config.CompileOptions
+    let dummyPaketDependencyManagerOption =
+        let currentDir = Path.GetDirectoryName typeof<Marker>.Assembly.Location
+        sprintf "--compilertool:\"%s\"" currentDir
+
     // see https://github.com/fsharp/FSharp.Compiler.Service/issues/755
     // see https://github.com/fsharp/FSharp.Compiler.Service/issues/799
     let options =
         { co.FsiOptions with
             FullPaths = true
-            ScriptArgs = "--simpleresolution" :: "--targetprofile:netstandard" :: "--nowin32manifest" :: "-o" :: outDll :: context.Config.ScriptFilePath :: co.FsiOptions.ScriptArgs
+            ScriptArgs = "--simpleresolution" :: "--targetprofile:netstandard" :: "--nowin32manifest" :: "--langversion:preview" :: dummyPaketDependencyManagerOption :: "-o" :: outDll :: context.Config.ScriptFilePath :: co.FsiOptions.ScriptArgs
```

Comment:
`preview` is not breaking (and will not be breaking in the future), correct?
The problem is: If this is breaking how can we rollout the change. Do we need to release `FAKE 6-alpha`?

Reply:
here's the set of features that opting in to preview brings:


```
// F# preview
LanguageFeature.NameOf, previewVersion
LanguageFeature.OpenStaticClasses, previewVersion
LanguageFeature.PackageManagement, previewVersion
LanguageFeature.AndBang, previewVersion
LanguageFeature.NullableOptionalInterop, previewVersion
```

When package management goes 'stable' we can change this langversion flag to point to `5.0` instead and 'pin' at that version for compatibility. This _shouldn't be breaking, because each of these features are additive:

* nameof just makes nameof available (it wasn't before)
* open static are a new feature that couldn't be done before
* ditto for package management
* and! only matters for CEs that have it implemented (I'd wager this is uncommon in FAKE scripts)
* similar points for nullable/option interop enhancements.

Reply:
> But in general I'd want to get off of preview as soon as possible.

Or let the user decide somehow (which I think he can already by adding the corresponding option himself)

In any case I don't think we have a lot of a choice anyway.

Reply:
note for posterity: I think we should be able to sidestep the entire dependencymanager feature if we _don't_ specify langversion:preview. IE the current error behavior is a bug for 'normal' consumers IMO. Discussion is ongoing on dotnet/fsharp about how this should be handled, so we may be reverting this change in the relatively short term :)